resources/js/processes/screens/index.js

```diff
@@ -16,6 +16,12 @@ new Vue({
     screenModal: false,
     screenId: null,
   },
+  mounted() {
+    const searchParams = new URLSearchParams(window.location.search);
+    if (searchParams.size > 0 && searchParams.get("create") === "true") {
+      this.$root.$emit("bv::show::modal", "createScreen", "#createScreenModalButton");
+    }
+  },
```

Comment:
Using URL queries to establish that I want to create a new screen and the modal should open automatically